### PR TITLE
Starting to Process Bullet Point 1 of Issue #6

### PR DESCRIPTION
**Pull request by kittysocks**

### CODE REVIEW

File: scripts/00_merge_selected_models_output.R [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+## In the end, this script should (or change it an Rmd!!)
+# read in the "../results/selected_models_output/" files and merge to 1 csv. SAVE IT for future use.
+# then, make tibbles ala csv_tidydata_progress. Specifically:
+#### 1) number of models per dataset
+#### 2) we will want to know whether each alignment's model == / != #50's model
+
+# Define Paths --------------------------
+csv_directory <- "../results/selected_models_output/"
+csv_path <-"../results/all_selected_models_.csv"
```
Comment (sjspielman) [inline scripts/00_merge_selected_models_output.R:18]: too many underscores?

File: scripts/00_merge_selected_models_output.R [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+# at this point looks like:
+#filename,name,datatype,AIC,AICc,BIC
+#EMGT00050000000002.Drosophila.001_AA_bestmodels.csv,EMGT00050000000002.Drosophila.001,AA,Dayhoff+F+I+G4,Dayhoff+F+I+G4,Dayhoff+F+G4
+
+# we want:
+#name,datatype,species,AIC,AICc,BIC
+#EMGT00050000000002,AA,Drosophila,Dayhoff+F+I+G4,Dayhoff+F+I+G4,Dayhoff+F+G4
+write_csv(csvdf,path=csv_path)
+
+# Read in Newly Created CSV ---------------------
```
Comment (sjspielman) [inline scripts/00_merge_selected_models_output.R:39]: STOP HERE! Going forward should be a new script.

**Comment by sjspielman** [comment]
General note: Name rmarkdown files `.Rmd`, NOT `.rmd`.

File: scripts/01_calculate_models_per_dataset.R [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  dplyr::select(-filename,-STOP) %>%
+  pivot_longer(AIC:BIC, names_to = "ic_type", values_to = "model")-> processed_00_models
+
+# How Many Models Per Dataset?--------------------------------
+
+process_ready_data <- function(dataset,a_species,a_datatype)
+{
+  dataset %>%
+    filter(species==a_species,datatype==a_datatype)%>%
+    group_by(name,model,ic_type,datatype) %>%
```
Comment (sjspielman) [inline scripts/01_calculate_models_per_dataset.R:31]: `count()` is a shortcut for :
`group_by() %>% tally() %>% ungroup()`

File: scripts/01_calculate_models_per_dataset.R [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+# Wrangle Header Data ------------------------
+
+csv_dataframe %>%
+  separate (name,c("name","species","STOP"),sep="[.]") %>%
+  dplyr::select(-filename,-STOP) %>%
+  pivot_longer(AIC:BIC, names_to = "ic_type", values_to = "model")-> processed_00_models
+
+# How Many Models Per Dataset?--------------------------------
+
+process_ready_data <- function(dataset,a_species,a_datatype)
```
Comment (sjspielman) [inline scripts/01_calculate_models_per_dataset.R:27]: maybe, `calculate_num_models_per_dataset `?

**Comment by kittysocks** [comment]
Let's Organize README in scripts/ 
Go into 01, and do high level statement of things we're doing 
Finalize these three scripts (00, 01, 02) 
use this information in conjunction with the scores
Redefine the paths and csv variables -> change it to path of specific file, doesn't necessarily need to be called a "csv" 
outside of scripts, make a directory called plots 
most common model percentage -> save this information, each script should be producing something for use in the next step 
best thing to do -> work with these three files then clean and put them through

potentially start writing, but write in pen and paper
write down what you're doing, stare at it and make connections with things you're doing

**Pull request closed without merging**